lti_consumer/migrations/0005_migrate_keyset_to_model.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.2.16 on 2020-10-24 11:47
+
+from django.db import migrations, models, transaction
+
+
+def _load_block(location):
+    """
+    Loads block from modulestore.
+    """
+    from xmodule.modulestore.django import modulestore
+    return modulestore().get_item(location)
+
+
+def forwards_func(apps, schema_editor):
+    """
+    Forward migration - copy client ID and save it in Model
+    """
+    LtiConfiguration = apps.get_model("lti_consumer", "LtiConfiguration")
+    db_alias = schema_editor.connection.alias
+
+    lti_configs = LtiConfiguration.objects.select_for_update().filter(
+        version="lti_1p3",
+    )
+    with transaction.atomic():
+        for lti_config in lti_configs:
+            block = _load_block(lti_config.location)
+
+            # If client_id exists, move it to model
+            if block.lti_1p3_client_id:
+                lti_config.lti_1p3_client_id = block.lti_1p3_client_id
+
+            # If key exists, move it to model. Set kid as client_id, respecting
+            # old implementation
+            if block.lti_1p3_block_key:
+                lti_config._lti_1p3_platform_private_key_id = block.lti_1p3_client_id
```

Comment:
Please confirm that you want to use `block.lti_1p3_client_id` as the value for this. You are checking above whether the block has the `lti_1p3_client_id` before setting it on the config, which means you expect it to be possible that it isn't present. You subsequently use it for the `_lti_1p3_platform_private_key_id` without verifying it is present.

Reply:
In the current `master` implementation, we're `client_id` and `key_id` are the same variable. I want to create a separate ID for each, but keep the old values for backwards compatibility. 

Does that address your concern?

>You subsequently use it for the _lti_1p3_platform_private_key_id without verifying it is present.

It was supposed to be a nested if. Will fix it :)